### PR TITLE
Add dockerfiles

### PR DESCRIPTION
**Pull request by stlaz**
/assign @tnozicka

### CODE REVIEW

File: images/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM registry.svc.ci.openshift.org/openshift/builder:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/template-service-broker
+COPY . .
+RUN make build --warn-undefined-variables
+
+FROM centos:7
+COPY --from=builder /go/src/github.com/openshift/template-service-broker/template-service-broker /usr/bin/
+ENTRYPOINT ["/usr/bin/template-service-broker"]
+LABEL io.k8s.display-name="OpenShift Template Service Broker" \
+      io.k8s.description="This is a component of OpenShifti, it gives the service catalog visibilityof the default Instant App and Quickstart templates" \
```
Comment (tnozicka) [inline images/Dockerfile]: typo

File: images/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM registry.svc.ci.openshift.org/openshift/builder:golang-1.12 AS builder
+WORKDIR /go/src/github.com/openshift/template-service-broker
+COPY . .
+RUN make build --warn-undefined-variables
+
+FROM centos:7
+COPY --from=builder /go/src/github.com/openshift/template-service-broker/template-service-broker /usr/bin/
+ENTRYPOINT ["/usr/bin/template-service-broker"]
+LABEL io.k8s.display-name="OpenShift Template Service Broker" \
+      io.k8s.description="This is a component of OpenShift, it gives the service catalog visibilityof the default Instant App and Quickstart templates" \
```
Comment (tnozicka) [inline images/Dockerfile]: visibilityof

**Comment by tnozicka** [comment]
(typo as follow up)

File: images/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+COPY . .
+RUN make build --warn-undefined-variables
+
+FROM centos:7
+COPY --from=builder /go/src/github.com/openshift/template-service-broker/template-service-broker /usr/bin/
+ENTRYPOINT ["/usr/bin/template-service-broker"]
+LABEL io.k8s.display-name="OpenShift Template Service Broker" \
+      io.k8s.description="This is a component of OpenShift, it gives the service catalog visibility of the default Instant App and Quickstart templates" \
+      com.redhat.component="template-service-broker" \
+      name="openshift/ose-template-service-broker" \
```
Comment (sttts) [inline images/Dockerfile]: is this name correct? Does it matter?

Comment (tnozicka) [reply images/Dockerfile]: I am not sure we should carry that name at all, that differs by distribution

**Pull request merged**